CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 ### Changed
 
 - `agate/Heading` to support marquee
+- `agate/TooltipDecorator` prop `tooltipMarquee` to support marquee for long tooltip and prop `tooltipWidth` to set a width for tooltip text
```

Comment:
It should be on Added not Changed.
And I think we need 
`agate/TooltipDecorator` prop `marquee` to ..... 
also.